js/plugins/vertexai/src/model_garden.ts

```diff
@@ -29,6 +29,23 @@ export const ModelGardenModelConfigSchema = OpenAIConfigSchema.extend({
   location: z.string().optional(),
 });
 
+export const llama32 = modelRef({
+  name: 'vertexai/llama-3.2',
+  info: {
+    label: 'Llama 3.2',
+    supports: {
+      multiturn: true,
+      tools: true,
```

Comment:
Are tools working?

Reply:
Wasn't able to test on Dev UI, did not test manually but I believe it should be supported in the model. Let me double check.